packages/editor/src/components/list-view-sidebar/style.scss

```diff
@@ -22,11 +22,21 @@
 	// This allows items in the list view to have equidistant padding left and right
 	// right up until a scrollbar is present.
 	scrollbar-gutter: auto;
+}
 
-	// The table cells use an extra pixels of space left and right. We compensate for that here.
+// The table cells use an extra pixels of space left and right. We compensate for that here.
+.editor-list-view-sidebar__list-view-panel-content {
+	// Keep the original padding for the List View panel so its spacing matches trunk.
 	padding: $grid-unit-05;
 }
 
+// Apply the new padding only to the Document Outline panel.
```

Comment:
These comments will be hard to interpret outside the context of this PR. Once this is merged, "new" and "old" padding won't be meaningful references anymore, so we should avoid adding this level of detail in the source. In general, unless there’s something truly non-obvious, we try to keep CSS comments minimal.